nikola/filters.py

```diff
@@ -160,36 +160,36 @@ def jpegoptim(infile):
     return runinplace(r"jpegoptim -p --strip-all -q %1", infile)
 
 
-def html_tidy_withconfig(infile):
+def html_tidy_withconfig(infile, executable=r'tidy5'):
```

Comment:
So should I remove the `r` prefix for all strings in all tidy5 related functions?